projects/elife.yaml

```diff
@@ -59,6 +59,12 @@ defaults:
             # external hdd
             size: 10 # GB
             device: /dev/sdh
+        elb:
+            # elb defaults only used if an 'elb' section present in project
+            stickiness: False
+            protocol: http
+            idle_timeout: 60
+            certificate: ASCAIRXYIRFBOR5QSDP5M
```

Comment:
perhaps give this a comment in the defaults?

`# id for ... ll: ABCDEFGHIJ123ABC5M`